### DISCUSSION TITLE
Writer to have multiple input sources.

### DISCUSSION THREAD

**Discussion by LuposX** [osmcode/pyosmium#223] [Q&A] [answered]
One useful use case for the writer would be to split up a big .pbf file into many more smaller .pbf files, in order for that to work, it would need to be possible to specify multiple output files in the writer and to specify inside the writer in which file the node/way should be added.
Alternatively does someone know a libary which would support sth. like that?

**Comment by lonvia** [comment, marked as answer]
You can simply have multiple writers open in parallel, each for a different file, and then choose for each node/way/relation the appropriate writer.
